docs/cache/manage/pricing.md

```diff
@@ -52,7 +52,7 @@ Heck yeah! You are billed for the inbound and outbound data transfer, not the vo
 
 <details>
   <summary>Will it scale?</summary>
-Hell yes! Momento services are the best way to future-proof your application story. You can add Momento Cache *extremely* quickly, and it just fades into the background whether you are doing 1 RPS or 1 million RPS. You just pay for what you use, so you get a scalable cache without spending a lot of cash.
+Hell yes! Momento services are the best way to future-proof your application story. You can add Momento Cache *extremely* quickly, and it just fades into the background whether you are doing 1 OPS or 1 million OPS. You just pay for what you use, so you get a scalable cache without spending a lot of cash.
```

Comment:
same here I think OPS is bit confusing vs TPS or RPS would just just `ops/s` to anotate